src/ui/components/cards/cardcomponent.js

```diff
@@ -64,6 +64,9 @@ export default class CardComponent extends Component {
      * @type {string}
      */
     this.verticalKey = data.verticalKey;
+
+    // additional field(s) for dataMappings config option
+    this.result._raw.distance = this.result.distance;
```

Comment:
I think this will error out if data.result is undefined, since in that case this.result will be set to `{}`

Reply:
ah yup, you are right. Updated!